### PR TITLE
Add multi-query box to the visualization page

### PR DESCRIPTION
**Pull request by bbrewer-papa-x**
smartcitiesdata/discovery_ui#177

co-authored-by: Scott Millard <smillard@hntb.com>

### CODE REVIEW

File: src/components/dataset-query/dataset-query.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import './dataset-query.scss'
+
+const DatasetQuery = props => {
+  const defaultText = `SELECT * FROM ${props.systemName}\nLIMIT 20000`
```
Comment (ManApart) [inline src/components/dataset-query/dataset-query.js]: Why place the limit on a new line?

Comment (smillardHNTB) [reply src/components/dataset-query/dataset-query.js]: Preference, trying to follow some sort of standard SQL styling.  Also makes it more obvious to the user IMO.

**Comment by smillardHNTB** [comment]
> Have you taken a look at: https://github.com/plotly/react-plotly.js#refreshing-the-plot?
> 
> Seems relevant to the issue with updating plot data.

We did, those things are not exposed by the editor so we don't have direct access to them.

**Comment by bbrewer-papa-x** [comment]
> Have you taken a look at: https://github.com/plotly/react-plotly.js#refreshing-the-plot?
> 
> Seems relevant to the issue with updating plot data.

I just did a quick test to see if updating this revision actually makes a difference and it does not. The real key is that we're never updating the `data` property of the plotly component based on datasources update.

**Pull request merged**